sql/hive/src/main/scala/org/apache/spark/sql/hive/MetastoreRelation.scala

```diff
@@ -142,6 +140,27 @@ private[hive] case class MetastoreRelation(
     }
   )
 
+  private def getSize(f: Path): Long = {
```

Comment:
I see what you're doing but this method no longer really does what it says. I think this is going to be error-prone.